src/components/apps/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'preact/hooks';
+import clsx from 'clsx';
+import { AppIcon } from '__/components/utils/AppIcon';
+import css from './Calendar.module.scss';
+import { YearView } from './Views/YearView';
+import { WeekView } from './Views/WeekView';
+import { MonthView } from './Views/MonthView';
+import { DayView } from './Views/DayView';
+import dayjs from 'dayjs';
+import { CalendarAppContext } from './context';
+import { mdiChevronLeft, mdiChevronRight } from '@mdi/js';
+
+enum VIEW_OPTIONS {
+  YEAR,
+  MONTH,
+  WEEK,
+  DAY,
+}
+
+export const Calendar = () => {
+  const [view, setView] = useState(VIEW_OPTIONS.MONTH);
+  const [selectedDate, setSelectedDate] = useState(dayjs());
+
+  const goToday = () => {
+    setSelectedDate(dayjs());
+  };
+
+  const goPrevMonth = () => {
+    setSelectedDate(selectedDate.add(-1, 'month'));
+  };
+
+  const goNextMonth = () => {
+    setSelectedDate(selectedDate.add(1, 'month'));
+  };
+
+  return (
+    <CalendarAppContext.Provider value={{ selectedDate }}>
+      <section class={css.container}>
+        <header class={clsx('app-window-drag-handle', css.titleBar)}></header>
+
+        <section class={css.mainArea}>
+          <div className={css.calendarHeader}>
+            <div>
+              <span className={css.month}>{selectedDate.format('MMMM')}</span>{' '}
+              <span className={css.year}>{selectedDate.format('YYYY')}</span>
+            </div>
+            <div className={css.controlButtons}>
+              <button onClick={goPrevMonth}>
```

Comment:
Also, you could use the `ButtonBase` component here. It doesn't have any styles to it, so overriding it will require less CSS.

Totally optional. Feel free to resolve this if you feel your approach is better.

Reply:
Ah. Nice way. Done migration to `ButtonBase`